src/api/apiCore.cpp

```diff
@@ -256,6 +256,15 @@ void TapeCore::tape_mesh_update_vertices(const YAML::Node& yamlNode, PlaybackSta
 	                         yamlNode[2].as<int32_t>());
 }
 
+rgl_status_t rgl_is_mesh_valid(rgl_mesh_t mesh, bool* out_valid)
+{
+	auto status = rglSafeCall([&]() {
+		*out_valid = Mesh::instances.find(mesh) != Mesh::instances.end();
+	});
+	TAPE_HOOK();
+	return status;
```

Comment:
- Added `CHECK_ARG` to ensure RGL fails in a controllable manner if the incoming pointer is null.
- Passed API call arguments to `TAPE_HOOK`. Tape is our home-baked solution to record all API calls to a file and then replay them from a file. It was created to eliminate the need to run the original application (often being a heavy game engine) when debugging RGL.
- Since we're using C++20, we can use `std::unorderd_map::contains(...)`
```suggestion
	auto status = rglSafeCall([&]() {
		CHECK_ARG(out_valid != nullptr);
		*out_valid = Mesh::instances.contains(mesh);
	});
	TAPE_HOOK(mesh, out_valid);
	return status;
```